docker/.gitignore

```diff
@@ -1,3 +1,5 @@
 cke
+cke-localproxy
 ckecli
+compile_resources
```

Comment:
`compile_resources` is built by `make install` indeed but not used in the form of a file.
Installing `compile_resources` as a part of `make install` seems excessive.